README.md

```diff
@@ -63,6 +63,7 @@ Server code is located at [ottomated/CrewLink-server](https://github.com/ottomat
 This is an example of how to list things you need to use the software and how to install them.
 * [Python](https://www.python.org/downloads/)
 * [node.js](https://nodejs.org/en/download/)
+* [Visual Studio](https://visualstudio.microsoft.com/downloads/) (Windows)
```

Comment:
@bmoe24x here is where you reference Visual Studio. Sure, Visual Studio Code is on there, but it's below the fold for me.

Reply:
I am aware that I added it as dependency, but not for the IDE capability. `yarn` was complaining it needed Visual Studio as a dependency

Reply:
What do you mean by "dependency"? I don't see any changes to the dependencies in `package.json`. Do you just mean how it's under the "Prerequisites" section? I guess yeah, technically VS Code is not required to edit this. People can use whatever editor they want.

Reply:
@cinderblock If you try to run `yarn dev` on a Windows machine, it fails with the following error:

```
gyp ERR! find VS
gyp ERR! find VS **************************************************************
gyp ERR! find VS You need to install the latest version of Visual Studio
gyp ERR! find VS including the "Desktop development with C++" workload.
gyp ERR! find VS For more information consult the documentation at:
gyp ERR! find VS https://github.com/nodejs/node-gyp#on-windows
gyp ERR! find VS **************************************************************
gyp ERR! find VS
```

This is why he had added the `Visual Studio` dependency, although admittedly it wasn't fully explained.

Reply:
Oh! That's a different issue.

The official line from Microsoft on how to setup your machine seems to be here: https://github.com/microsoft/nodejs-guidelines/blob/master/windows-environment.md#prerequisites

TL;DR, use [windows-build-tools](https://www.npmjs.com/package/windows-build-tools): `npm install --global windows-build-tools`

Alternatively, the manual way you described works.